src/hub/Hub.sol

```diff
@@ -114,7 +114,7 @@ contract Hub is Multicall, Auth, Recoverable, IHub, IHubGatewayHandler {
 
             totalPayoutShareAmount += payoutShareAmount;
             totalPaymentAssetAmount += paymentAssetAmount;
-            totalCancelledAssetAmount = cancelledAssetAmount;
+            totalCancelledAssetAmount += cancelledAssetAmount;
```

Comment:
Good one to make sure we cover in end-to-end tests in the next few weeks

Reply:
Although I believe technically this can never be set multiple times. Since you need to claim before submitting a new request after a cancelation.

Reply:
Might make sense to rename `totalCancelledAssetAmount`. @wischli let me know what you think

Reply:
Yeah good catch. It should be non-zero at most once, yes.